app/views/course/question_assessments/_question_assessment.json.jbuilder

```diff
@@ -12,6 +12,14 @@ json.unautogradable !question.auto_gradable? && assessment.autograded?
 json.type question_assessment.question.question_type_readable
 json.description format_ckeditor_rich_text(question.description) unless question.description.blank?
 
+is_programming_question = question.actable_type == Course::Assessment::Question::Programming.name
```

Comment:
convention is we only use camelCase if it's variable we want to expose to FE (i.e. whichever variable we describe as `json.xxx` we use camel case here, otherwise we use snake case)

Reply:
You're right, I saw the mistook the variable as json.xxx, marking as resolved